lib/rollbar/tasks/rollbar.cap

```diff
@@ -21,7 +21,12 @@ namespace :rollbar do
       request      = Net::HTTP::Post.new(uri.request_uri)
       request.body = ::JSON.dump(params)
 
-      Net::HTTP.start(uri.host, uri.port, :use_ssl => true) do |http|
+      p_env = ENV['http_proxy'] || ENV['HTTP_PROXY']
+      p_uri = p_env ? URI.parse(p_env) : nil
```

Comment:
This should be in a `begin/rescue` block to catch the `URI::InvalidURIError` that is possible to be thrown here. Furthermore, `p_env` truthy does not imply `p_uri` is not nil below, so you need to guard against that in the calls to `host` and `port`.